src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -294,13 +315,18 @@ private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId
         }
     }
 
+    /**
+     * This function trims the course ID if it is too long.
+     * @param demoCourseId
+     * @param maximumIdLength length to trim
+     * @return trimmed Course ID
+     */
     private String trimCourseIdToMaximumLengthIfNecessary(String demoCourseId, final int maximumIdLength) {
         final int courseIdLength = demoCourseId.length();
         if (courseIdLength <= maximumIdLength) {
             return demoCourseId;
         } else {
-            return demoCourseId.substring(courseIdLength
-                    - maximumIdLength);
+            return demoCourseId.substring(courseIdLength - maximumIdLength);
         }
     }
```

Comment:
`StringHelper` has already had `truncate()` function to do trimming. This function also trims the string but it keeps suffix part of the string while `truncate()` in `StringHelper` keeps prefix part. I think having 2 kinds of trimming functions in `StringHelper` could be misleading later. Besides this function is used in this class only so I doubt we should do that.